release/ray_release/glue.py

```diff
@@ -390,6 +392,7 @@ def run_release_test(
     cluster_env_id: Optional[str] = None,
     no_terminate: bool = False,
     test_definition_root: Optional[str] = None,
+    log_streaming_limit: int = LAST_LOGS_LENGTH,
```

Comment:
unit test for this new feature?

Reply:
added tests that the value passed into this function overrides the default value and go all the way to the cluster manager